WalletWasabi.Fluent/ViewModels/Wallets/Send/TransactionPreviewViewModel.cs

```diff
@@ -266,7 +266,7 @@ private async Task<bool> InitialiseTransactionAsync()
 
 			if (!result)
 			{
-				await ShowErrorAsync("Transaction Building", "At the moment, it is not possible to select a transaction fee that is less than the payment amount. The transaction cannot be sent.",
+				await ShowErrorAsync("Transaction Building", "At the moment, it is not possible to select a transaction fee that is more than the payment amount. The transaction could not be sent.",
```

Comment:
I would remove the first part. https://github.com/zkSNACKs/WalletWasabi/issues/7280#issuecomment-1038172059

```suggestion
				await ShowErrorAsync("Transaction Building", "It is not possible to select a transaction fee that is more than the payment amount. The transaction could not be sent.",
```